ChildMindInstitute/mindlogger-app-backend
# Add monthly events and "Show only on scheduled day" option (admin#304)

This PR adds support for monthly events and "Show only on scheduled day" option. It also refactors some of the schedule/notifications code in order to make it more readable.

[Issue admin#304](https://app.zenhub.com/workspaces/mindlogger-5e11094d0c26311588da9626/issues/childmindinstitute/mindlogger-admin/304)

**Testing this PR:**
1. Make sure you have the latest version of the admin.
2. Create a new event of "Monthly" type on the desired day of the month.

**Expected result:**
You should only receive notifications on the selected day of the month, and only on the months contained between the start date and end date.
